@here/harp-mapview/lib/MapView.ts

```diff
@@ -1234,6 +1234,26 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_visibleTileSetOptions.projection;
     }
 
+    /**
+     * Changes the projection at run time.
+     *
+     * TODO: There seems to be some issue with the sphere projection, when changing from this
+     * projection to a planar projection, the map is rotated. This needs to be fixed.
+     */
+    set projection(projection: Projection) {
+        // The geo center must be reset when changing the projection, because the
+        // camera's position is based on the projected geo center.
+        const geoCenter = this.geoCenter;
+        this.m_visibleTileSetOptions.projection = projection;
+        this.clearTileCache();
+        // We reset the theme, this has the affect of ensuring all caches are cleared.
+        this.theme = this.theme;
+        this.geoCenter = geoCenter;
+        // Necessary for the sphereProjection, however this also resets the camera position, so it
+        // should be fixed.
+        //this.setupCamera();
```

Comment:
Remove unnecessary comment

Reply:
I want to keep it, because it is necassary for the sphere projection (see the above TODO as well), I could create a ticket and put it here instead?

Reply:
@nzjony Got it, it's fine then